src/hooks.ts

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Errors, Todo } from './types';
+
+export const useGetTodos = (USER_ID: number, makeAnyChange: boolean) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<Errors>(Errors.noEroor);
```

Comment:
There seems to be a typo in 'Errors.noEroor'. It should probably be 'Errors.noError'. Please correct it to avoid confusion or possible bugs in the future.